libnmstate/schema.py

```diff
@@ -24,50 +24,50 @@
 
 def load(schema_name):
     return yaml.load(
-        pkgutil.get_data('libnmstate', 'schemas/' + schema_name + '.yaml'),
+        pkgutil.get_data("libnmstate", "schemas/" + schema_name + ".yaml"),
         Loader=yaml.SafeLoader,
     )
 
 
-ifaces_schema = load('operational-state')
+ifaces_schema = load("operational-state")
```

Comment:
ditto.

Reply:
I think 'black' has change this for me

Reply:
Please use `alias black='black -S -l 79'`
or in vimrc:

```
let g:black_linelength = 79
let g:black_skip_string_normalization = 1
```

Reply:
This is old stuff, now I follow that approach.